### ISSUE TITLE
Buffering issue on Firestick

### ISSUE THREAD

**Issue by Chris-Musume-The-WW** [yuliskov/SmartTube#2274]
Anyone having buffering/lagging issues on Smart-Tube TV for Firestick? Anytime I try to watch a video, it would lag for a few seconds or longer than that. It would lag constantly or randomly at times. Any fixes?

**Comment by ali-ee** [comment]
Yes, I have the same issue. Official app is fine. Tried changing buffer settings, didn't help.

**Comment by Chris-Musume-The-WW** [comment]
> Yes, I have the same issue. Official app is fine. Tried changing buffer settings, didn't help.

Man. Hopefully a update comes to fix this issue

**Comment by flashseal** [comment]
Chromecast 4k too

**Comment by GhoulUnlimited** [comment]
Same thing happens with Xiaomi Mi Box S

**Comment by t1ger5** [comment]
Same here.

On the Beta app I went to Settings - Video Player - Developer Options then turned on Force Legacy Codecs and Buffering Fix and it seemed to solve my issue.

It caps everything to 720p but that’s fine for my needs until there’s a fix

**Comment by Chris-Musume-The-WW** [comment]
> Same here.
> 
> On the Beta app I went to Settings - Video Player - Developer Options then turned on Force Legacy Codecs and Buffering Fix and it seemed to solve my issue.
> 
> It caps everything to 720p but that’s fine for my needs until there’s a fix

I tried that & it works as of now. Thanks for the info.

**Comment by jtkrause11** [comment]
I'm having the same issue on multiple Firesticks but YouTube Vanced works fine on my phone.
